Dockerfile

```diff
@@ -15,13 +15,27 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 FROM common AS sgx
+ARG PSW_VERSION=2.13.103.1
+ARG DCAP_VERSION=1.10.103.1
 RUN apt update && \
-    apt install -y --no-install-recommends libsgx-dcap-ql libsgx-dcap-ql-dev libsgx-dcap-quote-verify libsgx-enclave-common libsgx-urts az-dcap-client && \
+    apt install -y --no-install-recommends \
+    az-dcap-client \
+    libsgx-dcap-ql=$DCAP_VERSION-bionic1 \
+    libsgx-dcap-ql-dev=$DCAP_VERSION-bionic1 \
```

Comment:
libsgx-dcap-ql-dev was there before, but is it required or can we also move it to sgx-dev?

Reply:
Required for quote generation